### PR TITLE
Making the api call

### PR DESCRIPTION
**Pull request by Ariuka11**

### CODE REVIEW

File: github-usercard/src/App.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        this.setState({followers: res.data})
+      })
+      .catch(err => {
+        console.log('Not success', err)
+      })
+  }
+  render(){
+    return(
+      <div>
+        <UserCard {...this.state.profile} followers={this.state.followers} />
```
Comment (liam-mei) [inline github-usercard/src/App.js:35]: Interesting way of passing all the individual properties of the user object as a prop here.  I didn't think to do it that.  I would have preferred to pass the whole object as user={this.state.profile} or just pass the 3 properties that you're using.

File: github-usercard/src/components/UserCard.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+class UserCard extends React.Component{
+    render() {
+        return (
+            <div>
+                {this.props.name}
```
Comment (liam-mei) [inline github-usercard/src/components/UserCard.js:7]: The pieces here are erroring out for me for some reason.  I don't know why.  I'll have to have you show me the app

**Review by liam-mei — APPROVED** [review]

**Comment by liam-mei** [comment]
## MVP
- [x] componentDidMount to fetch user data/followers
- [ ] Display Data ???
- [ ] style user card

## Stretch
- [ ] Form to search github
- [ ] github contribution graph

------------
score: 1
------------

Good job using all class components in the project.  I would have liked to see more styling in the app though. I'm having some issues viewing the project on gitpod, I'll have to take a look at it when you demonstrate it to me.

**Comment by Ariuka11** [comment]
So sorry i just finished designing and made some changes on the followers.

**Pull request merged**